### PR TITLE
Add support for reading tags from a story file

### PR DESCRIPTION
**Pull request by benferse**
Addresses #1

### CODE REVIEW

**Review by bengreenier — APPROVED** [review]

File: demo.js [hunk 1]
```diff
@@ -18,3 +18,5 @@ const run = async () => {
 };
 
 run();
+
+console.log(JSON.stringify(parseHTML(getDemoData()), null, 2));
```
Comment (unwitting) [inline demo.js]: Maybe remove this to keep the demo from getting cluttered with dev lines.

**Review by unwitting — COMMENTED** [review]
One comment to remove something from `demo.js`, beyond that it looks good!

File: demo.js (hunk 1 above)
Comment (benferse) [reply demo.js]: Done!

**Comment by unwitting** [comment]
@benferse @bengreenier this has been published to npm as version `1.3.0`.

**Pull request merged**